src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81</PackageTargetFallback>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <Company>Autofac Project - http://autofac.org</Company>
+    <Product>Autofac</Product>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseUrl></PackageLicenseUrl>
+    <Copyright>Copyright © 2017 Autofac Contributors</Copyright>
+    <Description>MvvmCross integration for Autofac IoC</Description>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="autofac" Version="4.6.1" />
+    <PackageReference Include="MvvmCross.Core" Version="5.1.1" />
```

Comment:
Why the update to MvvmCross 5? Did you make changes that aren't compatible with 4 and would force consumers to take an upgrade?